esx_service/utils/vmdk_utils.py

```diff
@@ -65,14 +67,27 @@ def get_datastores():
 
     return datastores
 
-def get_volumes():
+def get_volumes(tenant_re, non_tenant_mode):
```

Comment:
Instead of a boolean parameter `non_tenant_mode`, I would just check to see if `tenant_re` is None. This provides the same amount of information but doesn't require the second argument at all.